pages/mesosphere/dcos/version-policy/index.md

```diff
@@ -376,7 +376,7 @@ The designations are as follows:
         <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Elastic 2.7.x-6.8.1 (Recommended)</td>
```

Comment:
we should update [this line](https://github.com/mesosphere/dcos-docs-site/pull/2630/files#diff-cfd7625821cec803487a378a7d3e228cL373) so that we publicly state we support 
- Elastic 6.8.1
- Elastic 7.4.1